app/views/admin/shared/_sidenav_product.html.haml

```diff
@@ -1,8 +1,8 @@
 %ul.sidebar-nav
-  = tab "#{Instrument.model_name.human.pluralize} (#{current_facility.instruments.count})", facility_instruments_path(current_facility), sidenav_tab == Instrument.model_name.plural
-  = tab "#{Service.model_name.human.pluralize} (#{current_facility.services.count})", facility_services_path(current_facility), sidenav_tab == Service.model_name.plural
-  = tab "#{Item.model_name.human.pluralize} (#{current_facility.items.count})", facility_items_path(current_facility), sidenav_tab == Item.model_name.plural
-  = tab "#{Bundle.model_name.human.pluralize} (#{current_facility.bundles.count})", facility_bundles_path(current_facility), sidenav_tab == Bundle.model_name.plural
+  - [Instrument, Service, Item, Bundle].each do |product_type|
```

Comment:
I'm probably going to pull this into some sort of registry once I have a new product type. I wanted to use `Product.descendents`, but that's unreliable: if the subclass hasn't been used yet (and thus not auto-loaded), it's not there, and there's no way to designate order.